### DISCUSSION TITLE
I have a model I created with RVC (Retrieval-based-Voice-Conversion-WebUI), can I use it with Coqui TTS? How?

### DISCUSSION THREAD

**Discussion by brunolm** [coqui-ai/TTS#2755] [General Q&A]
I have a model I created with RVC (Retrieval-based-Voice-Conversion-WebUI). I have the `.pth` file in the `weights` folder.

Is it possible to use the same file on Coqui TTS? How?

**Comment by erogol** [comment]
its not possible. You can only use models that are implemented in TTS
